README.md

```diff
@@ -1,161 +1,720 @@
+>
+> ― [Deleuze] & [Guattari], [A Thousand Plateaus: Capitalism and Schizophrenia] 
+
 ## 1.1 Motivation
 
-While existing databases increasingly treat network partitions as unavoidable disturbances of a network, many mitigate these disturbances under a model that either assumes they'll be bounded in length, or by allowing them to impact the availability of some nodes in the network.
+While existing databases increasingly treat network partitions as unavoidable disturbances to uptime, many mitigate these situations under a model that either assumes they will be bounded in length, or by allowing them to have some or all nodes cease operation. Such techniques are unsuitable for far-edge and local-first applications, where disorder is the norm, network partitions are ubiquitous and unbounded, and there's an uncompromising need for availability.
+
+Modern environments increasingly involve unstable dynamic network topologies of heterogenous peers for which common definitions of consistency often do not apply. In this context, continued operation is not just desirable, but enforcing continuous consistency between all peers is a meaningless proposition. This begs the question: what if there was [no concept of "primary site"][A Certain Tendency of the Database Community] and all data were considered subjective relative to the viewer.
+
+## 1.2 Approach
+
+> No single theory ever agrees with all the facts in its domain.
+> 
+> — [Paul Feyerabend], [Against Method]
+
+PomoDB addresses the above constraints with query semantics that guarantee eventual consistency between all peers with access to the relevant data, but never requires nor assumes full synchronization between clients. This enables PomoDB to act as a sound foundation for globally distributed data with an indeterminate number of transient peers, with varied access patterns, and ever changing access to data.
 
-Such techniques are unsuitable for far-edge and local-first applications, where disorder is the norm, network partitions are ubiquitous and unbounded, and there's an uncompromising need for availability. These environments also often involve dynamic network topologies made up of heterogenous peers, for which common definitions of consistency may not apply.
+Historically databases have assumed an objective worldview: there is a single, correct, _objective_ way to look at data. This can be convenient, but is also overly simplistic for many cases. As application state becomes increasingly [distributed][Fallacies of distributed computing], partition-tolerant, private, and remixable, modeling data as _subjective_ makes increasing sense. This is a [postmodern worldview][Postmodernism], which recognizes that there is no one true way to look at things, but rather multiple interpretations that are useful for different purposes (including [mathematics][Pomo Math] and science). The only objective facts in PomoDB are the quads (as statements) themselves, and their claimed [causal ordering][B-Series].
```

Comment:
> The only objective facts in PomoDB are the quads (as statements) themselves

I know what this means, and I agree, but I wonder if it isn't clear without our background, since this reads like it's saying that the quads are objectively true, when it's really "that the quads have been asserted with some given causal ordering" that's objectively true.

Reply:
Updated:

```diff
- The only objective facts in PomoDB are the quads
- (as statements) themselves, and their claimed [causal ordering][B-Series].
+ The only objective facts in PomoDB are that the quads
+ themselves were asserted by their author, and their claimed [causal ordering][B-Series].
```